.travis.yml

```diff
@@ -8,6 +8,7 @@ python:
   - 3.7
   - 3.8
   - 3.9
+  - 3.10
```

Comment:
We no longer use Travis. Please delete this file and use github actions instead.